capa/helpers.py

```diff
@@ -238,3 +240,64 @@ def is_running_standalone() -> bool:
     # so we keep this in a common area.
     # generally, other library code should not use this function.
     return hasattr(sys, "frozen") and hasattr(sys, "_MEIPASS")
+
+
+def is_dev_environment() -> bool:
+    if is_running_standalone():
+        return False
+
+    if "site-packages" in __file__:
+        # running from a site-packages installation
+        return False
```

Comment:
Should we also consider `dist-packages` here?

Reply:
![image](https://github.com/mandiant/capa/assets/156560/8c00faeb-d009-4137-ac2b-935950f4f59b)

Reply:
doesn't *seem* important but if you can demonstrate it's relevant this shouldn't be hard